cpp/test/IceDiscovery/simple/Client.cpp

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) ZeroC, Inc.
 
 #include "Ice/Ice.h"
-#include "Ice/RegisterPlugins.h"
 #include "Test.h"
 #include "TestHelper.h"
 
+// Link with IceDiscovery on Windows.
+#if defined(_MSC_VER) && !defined(ICE_DISABLE_PRAGMA_COMMENT)
```

Comment:
do we use ICE_DISABLE_PRAGMA_COMMENT with the tests?

Reply:
I don't think we do, it's for consistency. Whenever we use pragma comment, we also use this macro.